src/main/resources/com/google/api/codegen/java/test.snip

```diff
@@ -272,9 +272,15 @@
 
   @join assert : test.asserts
     @if assert.hasExpectedValueTransformFunction
-      Assert.assertEquals({@assert.expectedValueTransformFunction}(\
-        {@assert.expectedValueIdentifier}), \
-        actualRequest.{@assert.actualValueGetter}());
+      @if assert.hasActualValueTransformFunction
```

Comment:
We don't handle case of having an actualValueTransform but not an expectedValueTransform. That seems fine if it isn't used, but we should at least comment that this case is unhandled